pkgs/by-name/co/contact/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "contact";
+  version = "1.3.16";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "pdxlocations";
+    repo = "contact";
+    rev = version;
+    hash = "sha256-Fezjvs+aP2Qlp0A6P4/5QogVOHkxRaey9ZkI70va19Y=";
+  };
+
+  dependencies = [ python3Packages.meshtastic ];
+
+  build-system = [ python3Packages.poetry-core ];
+
+  meta = {
+    homepage = "https://github.com/pdxlocations/contact";
+    description = "Console UI for Meshtastic";
+    mainProgram = "contact";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [
+      sarcasticadmin
+    ];
```

Comment:
Should set the `platforms` attribute. I looks like it really only supports Linux and Darwin.
https://github.com/pdxlocations/contact/blob/fe9807558291b1e40ecc30f478d2c4265a28741f/contact/message_handlers/rx_handler.py#L36-L67